cmake/FindFFTS.cmake

```diff
@@ -0,0 +1,67 @@
+#[[
+MIT License
+
+Copyright (c) 2019 - 2025 Advanced Micro Devices, Inc.
```

Comment:
@r-abishek please change this block to
```
Copyright © Advanced Micro Devices, Inc., or its affiliates.

SPDX-License-Identifier:  [Identify the open source license]
```

Dates are not required anymore

Reply:
done